ui/generate-rcc.go

```diff
@@ -52,6 +52,9 @@ func main() {
 			if err != nil {
 				return err
 			}
+            if info.IsDir() && (info.Name() == "vendor" || info.Name() == "tests") {
+                return filepath.SkipDir
+            }
```

Comment:
This looks like this needs to run through a formatter

Reply:
How to do this? My first ever piece of `go` code, bear with me :)

Reply:
There's a formatting tool something like

```sh
gofmt -w yourcode.go
```

https://go.dev/blog/gofmt

Congrats on your first go code!

Reply:
Done